platform/mds/mds/src/test/java/org/motechproject/mds/it/osgi/MdsBundleIT.java

```diff
@@ -168,14 +183,45 @@ public void tearDown() throws Exception {
 
     @Test
     public void testEntitiesBundleInstallsProperly() throws Exception {
+
+        //Run tests with l2 cache active
+        testEntitiesBundleInstallsProperly(true);
+
+        //Turn off l2 cache
+        outputStream =  new FileOutputStream(configurationFile);
```

Comment:
Always close streams - you can use try with resource. Moreover I don't see a point the stream being stored as an instance variable.